decidim-admin/config/locales/ca.yml

```diff
@@ -105,6 +114,24 @@ ca:
         update:
           error: S'ha produït un error en l'actualització d'aquest procés participatiu.
           success: El procés participatiu s'ha actualitzat correctament.
+      static_pages:
+        actions:
+          view: Visualitzar la pàgina pública
+        create:
+          error: Error
```

Comment:
Alternatively, you can remove these keys and wait until #225 is merged and add these translations via Crowdin.